charts/hashr/templates/hashr-gcp-cronjob.yaml

```diff
@@ -0,0 +1,62 @@
+{{- if .Values.hashr.importers.gcp.enabled -}}
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: {{ .Release.Name }}-hashr-gcp
+spec:
+  schedule: {{ .Values.hashr.importers.gcp.schedule | quote }}
+  concurrencyPolicy: Forbid
+  successfulJobsHistoryLimit: 2
+  failedJobsHistoryLimit: 1
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          containers:
+          - name: hashr-gcp
+            image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+            imagePullPolicy: {{ .Values.image.pullPolicy }}
+            args:
+              - --logtostderr=1
+              - -storage
+              - postgres
+              - -exporters
+              - postgres
+              - -postgres_host
+              - {{ include "common.names.fullname" (dict "Chart" (dict "Name" "postgresql") "Release" .Release "Values" .Values.postgresql) }}
+              - -postgres_port
+              - {{ .Values.postgresql.primary.service.ports.postgresql | quote }}
+              - -postgres_user
+              - {{ .Values.postgresql.auth.username | quote }}
+              - -postgres_password
+              - "$(POSTGRES_PASSWORD)"
+              - -postgres_db
+              - {{ .Values.postgresql.auth.database | quote }}
+              - -importers
+              - GCP
+              - -gcp_projects
+              - {{ .Values.hashr.importers.gcp.gcp_projects | quote }}
+              - -hashr_gcp_project
+              - {{ .Values.hashr.importers.gcp.hashr_gcp_project | quote }}
+              - -hashr_gcs_bucket
+              - {{ .Values.hashr.importers.gcp.hashr_gcs_bucket | quote }}
+            env:
+            - name: GOOGLE_APPLICATION_CREDENTIALS
+              # Store your SA key in the hashrvolume/creds/ folder via "kubectl cp"!
```

Comment:
In the future we should consider using GCP workload identity which is the recommended method. That way no need to export keys and can use the `init-gke.sh` script to bootstrap the service account.